dev/breeze/src/airflow_breeze/utils/selective_checks.py

```diff
@@ -258,7 +258,7 @@ def find_all_providers_affected(
                 all_providers.add(provider)
     if all_providers_affected:
         return "ALL_PROVIDERS"
-    if suspended_providers:
+    if suspended_providers and fail_if_suspended_providers_affected:
```

Comment:
I don't think we should skip this block?

If there are suspended providers we should still print the messages in the block
The only effect the new flag should have is if we do `sys.exit(1)` or `break`

Reply:
I can move the if below and live the printing if you want. It will be the output of selective checks so you will usually not look at it, but well, I can add it. In fact I added a bit more colorful output to be a bit more verbose on what's happening.

There is no break to do here thouhg. The list of affected providers is built above already and it already does not contain suspended providers in `all_providers` variable, this has not changed , so there is nothing to `break` here.

Reply:
This is where the list was built: 

```
if provider not in dependencies:
    suspended_providers.add(provider)
else:
    all_providers.add(provider)
```